ci/travis/ci.sh

```diff
@@ -177,7 +177,8 @@ test_python() {
       -python/ray/tests:test_stress  # timeout
       -python/ray/tests:test_stress_sharded  # timeout
       -python/ray/tests:test_k8s_cluster_launcher
-      -python/ray/tests:test_k8s_operator_examples
+      -python/ray/tests/kubernetes_e2e:test_k8s_operator_examples
+      -python/ray/tests/kubernetes_e2e:test_k8s_operator_scaling
```

Comment:
I'm not sure about the syntax here. Is this correct?
Or, since these are in a subdirectory of tests do they need to be explicitly excluded here?

Reply:
Since these are not in tests/BUILD they don't need to be excluded.